### PR TITLE
Removes Certification V1 code paths, removes V1-specific specs

### PR DESCRIPTION
**Pull request by amprokop**

### CODE REVIEW

File: app/helpers/caseflow_form_builder.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class CaseflowFormBuilder < ActionView::Helpers::FormBuilder
+  # :nocov:
```
Comment (amprokop) [inline app/helpers/caseflow_form_builder.rb]: after removing tests that triggered this code, we get code coverage errors. don't want to make this PR too huge so will take care of this later:
https://github.com/department-of-veterans-affairs/caseflow/issues/3132

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 # rubocop:disable Metrics/ModuleLength
 module ApplicationHelper
+  # :nocov:
```
Comment (amprokop) [inline app/helpers/application_helper.rb]: after removing tests that triggered this code, we get code coverage errors. don't want to make this PR too huge so will take care of this later:
https://github.com/department-of-veterans-affairs/caseflow/issues/3132

File: app/helpers/form8_helper.rb [hunk 3]
```diff
@@ -1,7 +1,9 @@
 module Form8Helper
+  # :nocov:
```
Comment (amprokop) [inline app/helpers/form8_helper.rb]: after removing tests that triggered this code, we get code coverage errors. don't want to make this PR too huge so will take care of this later:
https://github.com/department-of-veterans-affairs/caseflow/issues/3132

**Comment by amprokop** [comment]
Thanks, @aroltsch!

**Comment by kierachell** [comment]
Failing in demo deploy:

```
ActiveRecord::RecordNotUnique: PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint \"index_users_on_station_id_and_css_id\"\nDETAIL:  Key (station_id, css_id)=(283, Reader) already exists.\n: INSERT INTO \"users\" (\"css_id\", \"station_id\", \"full_name\")
```

**Comment by amprokop** [comment]
@kierachell — that error is unrelated. I think we've seen it before, it might be a bug in the seed scripts.

**Pull request merged**